.isort.cfg

```diff
@@ -0,0 +1,8 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=True
+force_grid_wrap=0
+use_parentheses=True
+line_length=88
+known_first_party=_acurl
+known_third_party = aio_pika,altair,asyncmock,bs4,docopt,flask,ipdb,jinja2,mocks,msgpack,nanomsg,pandas,pkg_resources,psutil,pytest,pytest_httpserver,sanic,selenium,setuptools,ujson,uvloop,werkzeug,zmq
```

Comment:
What supreme laziness is this :'D

Reply:
isort is a package that sorts imports at the top of the file into sections: stdlib, third-party imports, and first-party (i.e. from the same package).  It tries to detect what is third-party and what is first-party, but it doesn't always do a good job at distinguishing.  So we specify in the config file which is which.

Speaking of supreme laziness, I did not write the config manually, but found a tool that does it automatically (otherwise I would not have bothered).  And that's all plumbed in as a pre-commit hook, so it will always stay up to date and I never have to think about it again.  That's the theory at least.  Supreme laziness to the max :P